packages/@aws-cdk/aws-amplify-alpha/lib/branch.ts

```diff
@@ -237,7 +237,7 @@ class AmplifyAssetDeploymentProvider extends NestedStack {
           __dirname,
           'asset-deployment-handler/index.js',
         ),
-        runtime: lambda.Runtime.NODEJS_14_X,
+        runtime: lambda.Runtime.NODEJS_16_X,
```

Comment:
Is this safe? Does Amplify not exist in GovCloud?

Shouldn't this be the same new floating runtime?